### PR TITLE
Update configure-auto-exposures.md

### PR DESCRIPTION
**Pull request by mirnawong1**
Add versionless as a prerequisite

## Checklist
- [ ] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] The topic I'm writing about is for specific dbt version(s) and I have versioned it according to the [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and/or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) guidelines.
- [ ] I have added checklist item(s) to this list for anything anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

File: website/docs/docs/collaborate/auto-exposures.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ Auto-exposures are currently available in beta to a limited group of users and a
 
 Auto-exposures helps users understand how their models are used in downstream analytics tools to inform investments and reduce incidents — ultimately building trust and confidence in data products. It imports and auto-generates exposures based on Tableau dashboards, with user-defined curation.
 
+Auto-exposures are available on [Versionless](/docs/dbt-versions/versionless-cloud) dbt.
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/auto-exposures.md]:
```suggestion
Auto-exposures is available on [Versionless](/docs/dbt-versions/versionless-cloud) and to [dbt Cloud Enterprise](https://www.getdbt.com/pricing/) plans.
```

File: website/docs/docs/cloud-integrations/configure-auto-exposures.md [hunk 2]
```diff
@@ -22,10 +22,11 @@ Auto-exposures help data teams optimize their efficiency and ensure data quality
...
-3. You have [admin permissions](/docs/cloud/manage-access/enterprise-permissions) in dbt Cloud to edit project settings or production environment settings
-4. Use Tableau as your BI tool and can enable metadata permissions or work with an admin to do so. Compatible with Tableau Cloud or Tableau Server with the Metadata API enabled. 
+1. Your environment and jobs are on [Versionless](/docs/dbt-versions/versionless-cloud) dbt.
+2. You have a dbt Cloud account on the [Enterprise plan](https://www.getdbt.com/pricing/).
+3. You have set up a [production](/docs/deploy/deploy-environments#set-as-production-environment) deployment environment for each project you want to explore, with at least one successful job run. 
+4. You have [admin permissions](/docs/cloud/manage-access/enterprise-permissions) in dbt Cloud to edit project settings or production environment settings
+5. Use Tableau as your BI tool and can enable metadata permissions or work with an admin to do so. Compatible with Tableau Cloud or Tableau Server with the Metadata API enabled.
```
Comment (mirnawong1) [inline website/docs/docs/cloud-integrations/configure-auto-exposures.md]:
```suggestion
5. Use Tableau as your BI tool and can enable metadata permissions or work with an admin to do so. Compatible with Tableau Cloud or Tableau Server with the Metadata API enabled. 
6. Run a production job _after_ saving the Tableau collections.
```

File: website/docs/docs/cloud-integrations/configure-auto-exposures.md [hunk 3]
```diff
@@ -22,10 +22,12 @@ Auto-exposures help data teams optimize their efficiency and ensure data quality
...
 
-1. You have a dbt Cloud account on the [Enterprise plan](https://www.getdbt.com/pricing/).
-2. You have set up a [production](/docs/deploy/deploy-environments#set-as-production-environment) deployment environment for each project you want to explore, with at least one successful job run. 
-3. You have [admin permissions](/docs/cloud/manage-access/enterprise-permissions) in dbt Cloud to edit project settings or production environment settings
-4. Use Tableau as your BI tool and can enable metadata permissions or work with an admin to do so. Compatible with Tableau Cloud or Tableau Server with the Metadata API enabled. 
+1. Your environment and jobs are on [Versionless](/docs/dbt-versions/versionless-cloud) dbt.
+2. You have a dbt Cloud account on the [Enterprise plan](https://www.getdbt.com/pricing/).
+3. You have set up a [production](/docs/deploy/deploy-environments#set-as-production-environment) deployment environment for each project you want to explore, with at least one successful job run. 
+4. You have [admin permissions](/docs/cloud/manage-access/enterprise-permissions) in dbt Cloud to edit project settings or production environment settings
```
Comment (nataliefiann) [inline website/docs/docs/cloud-integrations/configure-auto-exposures.md]:
```suggestion
4. You have [admin permissions](/docs/cloud/manage-access/enterprise-permissions) in dbt Cloud to edit project settings or production environment settings.
```

File: website/docs/docs/collaborate/auto-exposures.md [hunk 4]
```diff
@@ -16,6 +16,8 @@ Auto-exposures are currently available in beta to a limited group of users and a
 
 Auto-exposures helps users understand how their models are used in downstream analytics tools to inform investments and reduce incidents — ultimately building trust and confidence in data products. It imports and auto-generates exposures based on Tableau dashboards, with user-defined curation.
 
+Auto-exposures is available on [Versionless](/docs/dbt-versions/versionless-cloud) and to [dbt Cloud Enterprise](https://www.getdbt.com/pricing/) plans.
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/auto-exposures.md]:
```suggestion
Auto-exposures is available on [Versionless](/docs/dbt-versions/versionless-cloud) and on [dbt Cloud Enterprise](https://www.getdbt.com/pricing/) plans.
```

**Review by nataliefiann — COMMENTED** [review]
Hiya @mirnawong1 

Thanks for creating this PR! It was a good read. I committed a small change (added a full stop) to one of the sentences but other than that, I have no suggestions. I'm yet to approve this just in case you'd like Roxi to review this for you too but let me know and I'll approve this.

Kind Regards
Natalie

**Review by nataliefiann — APPROVED** [review]
Thanks @mirnawong1 this has been approved for you.

Kind Regards
Natalie

**Pull request merged**